util/newlibc.c

```diff
@@ -0,0 +1,26 @@
+#include <microkit.h>
+
+void _close(void)
+{
+    microkit_dbg_puts("NEWLIB ERROR: calling unimplimented _close()");
```

Comment:
Please fix spelling: 'unimplemented'

Reply:
Whoops, done. Added a comment at the top for the purpose of the file as well.